sympy/physics/control/lti.py

```diff
@@ -782,7 +884,21 @@ def var(self):
         p
 
         """
-        return self._var
+        return self.args[0].var
+
+    @property
+    def num_inputs(self):
+        return getattr(self.args[0], "num_inputs", None)
```

Comment:
Why would `self.args[0]` sometimes be an object that has this attribute and sometimes not?

That suggests a messy design somewhere.